tooling/internal/python/python.go

```diff
@@ -100,10 +107,21 @@ if _parse_version(_np.__version__) < _MIN_NUMPY_VERSION:
     raise ImportError(f"Your installed numpy version is {_np.__version__}, but version >= {'.'.join(str(i) for i in _MIN_NUMPY_VERSION)} is required.")
 `)
 
-	fmt.Fprintf(w, "from .yardl_types import *\n")
+	relativePath := ".."
+	if ns.IsTopLevel {
+		relativePath = "."
+	}
+	fmt.Fprintf(w, "from %syardl_types import *\n", relativePath)
+
+	for _, ref := range ns.GetAllChildReferences() {
+		fmt.Fprintf(w, "from %s import %s\n", relativePath, common.NamespaceIdentifierName(ref.Name))
+		fmt.Fprintf(w, "from %s%s.types import *\n", relativePath, common.NamespaceIdentifierName(ref.Name))
```

Comment:
Do we want to be able to program against referenced types using the imported "top-level" package? It can become a breaking change to introduce a type with the same name in the top level package as in an imported package if you've been programming against it using `toplevel.MyType` instead of `toplevel.imported.MyType`.

Reply:
I think it creates more room for user error. I will revert this so the user must access referenced types through their respective module.

However, in doing this, I discovered a bug in the Python codegen for a switch-case over union computed field (`writeSwitchCaseOverUnion`): The resolved union type is not namespaced.

Excerpt from generated `python/test_model/types.py`:

```
   def nested_switch(self) -> yardl.Int16:
        _var0 = self.union_with_nested_generic_union
        if isinstance(_var0, IntOrGenericRecordWithComputedFields.Int):
            return -1
        if isinstance(_var0, IntOrGenericRecordWithComputedFields.GenericRecordWithComputedFields):
            rec = _var0.value
            _var1 = rec.f1
            if isinstance(_var1, T0OrT1.T1):
                return int(20)
            if isinstance(_var1, T0OrT1.T0):
                return int(10)
            raise RuntimeError("Unexpected union case")
        raise RuntimeError("Unexpected union case")
```

`T0OrT1` should be `basic_types.T0OrT1`.

This one is harder to track down.